docs/index.rst

```diff
@@ -187,6 +187,11 @@ This command will generate a ``PostFactory`` class that looks like this:
         }
     }
 
+.. note::
```

Comment:
Can you move to the "Attributes" section below (right above the "Faker" title)

Reply:
Done.